src/commands/cronjob/create.tsx

```diff
@@ -97,18 +69,50 @@ export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
       throw new Error("no project found for app installation");
     }
 
+    let active: boolean = true;
+    if (disable) {
+      active = false;
+    }
+
+    let destination: CronjobCreationData["destination"];
+    if (url) {
+      destination = { url };
+    } else if (command && interpreter) {
+      let destinationInterpreter;
+      if (interpreter == "bash") {
+        destinationInterpreter = "/bin/bash";
+      } else {
+        destinationInterpreter = "/usr/bin/php";
+      }
+
+      destination = {
+        interpreter: destinationInterpreter,
+        path: command as string,
+      };
```

Comment:
This is largely duplicated with the same part from the `update` command. Maybe extract this into a common function (something like `function buildCronjobDestination(url: string|undefined, command: string|undefined, interpreter: string|undefined): CronjobCreationData["destination"]`)?